app/_src/gateway/breaking-changes/32x.md

```diff
@@ -142,4 +142,18 @@ Old parameter name | New parameter name
 `session_cookie_renew` | Removed, no replacement parameter. 
 `session_cookie_maxsize` | Removed, no replacement parameter. 
 `session_strategy` | Removed, no replacement parameter. 
-`session_compressor` | Removed, no replacement parameter. 
\ No newline at end of file
+`session_compressor` | Removed, no replacement parameter. 
+
+## Patch release breaking changes
```

Comment:
```suggestion
## Patches
```
Maybe let's just shorten to this. We're not supposed to say "patch releases" anymore.